delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/MainSuite.scala

```diff
@@ -27,6 +28,9 @@ import java.nio.file.{Files, Paths}
   */
 class MainSuite extends NexusSuite with MainSuite.Fixture {
 
+  // The default timeout of 30s is slightly too short for the GitHub free runners
+  override val munitIOTimeout: Duration = Duration(60, "s")
+
```

Comment:
Locally the test takes about 22s. On the cloud runner, it was timing out without this change, so must be >30s. However, even with this change, the workflow runs faster on the cloud runner than on the self-hosted one.

Reply:
Yep it is starting a ES node so it can take some time
It could me nice to cut this dependency on this test as it does not bring value here as we want to check configuration/ distage wiring here (it should be about the events metric IIRC)